src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -221,15 +221,17 @@ public final void createDatabase() throws SQLException, IOException {
      */
     public final void createDatabase(DatabaseType dbType, Connection conn) throws SQLException, IOException {
 
-            String ddlPath = this.getDatabaseDDLPath(dbType);
             ScriptRunner runner = new ScriptRunner(conn, true, true);
 
-            if (LOG.isDebugEnabled()) {
+            if (workConf.getDdlPath() != null) {
+                String ddlPath = workConf.getDdlPath();
```

Comment:
I would add a `WARN` message to indicate you are overriding the default DDL path.

Reply:
Added